pylinalg/obj/matrix.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+
+from .base import LinalgBase
+from ..func import (
+    matrix_compose,
+    matrix_decompose,
+    matrix_inverse,
+)
+
+
+class Matrix(LinalgBase):
+    """A representation of a transformation matrix in 4D
+    homogeneous (to 3D Euclidean) space."""
+
+    def __init__(self, matrix=None, /, *, dtype=None):
```

Comment:
I know that the star makes that all args that come after it are keyword-only. Do I understand correctly that the slash there makes that the args come before it position-only?

Reply:
Yes, that's correct